Client/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -782,6 +787,13 @@ extension NewTabPageViewController {
             }
         }
     }
+    
+    /// Moves New Tab Page Scroll to start of Brave Today - Used for shorcut
+    func scrollToBraveToday() {
```

Comment:
rename to `BraveNews`

Reply:
Fix at https://github.com/brave/brave-ios/pull/3579/commits/6094980027fe2d3ea410dd134ff093f8876e7a3c